src/util/fsUtil.js

```diff
@@ -30,4 +52,6 @@ module.exports = {
     path.dirname(filePathWithExt),
     path.basename(filePathWithExt, path.extname(filePathWithExt)),
   ),
+
+  copySyncWithOptions,
```

Comment:
Can just put the code for `copySyncWithOptions()` here directly, since other methods also do it the same way for this file, let's try to be consistent.